### DISCUSSION TITLE
Inconsistent Display of Unread Notifications in GitHub

### DISCUSSION THREAD

**Discussion by MartinDelille** [community/community#46348] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have observed that some GitHub notifications are not being displayed as unread, leading to missed updates. This issue has been occurring over the past few weeks.

The problem usually arises when I start a new issue, pull request or discussion. As an example, the response in the following link appeared in my notifications but was not marked as unread: https://github.com/cdelledonne/vim-cmake/issues/80#issuecomment-1416720496.

However, all other events related to this repository are displayed as unread.

I would appreciate it if someone could shed light on this issue and suggest a solution.

**Comment by jge162** [comment]
Clearing your browser cache and cookies may resolve the issue.

check your notifications settings, go to your GitHub profile, click on "Settings," then "Notifications." Make sure there is nothing changed there. 

internet connection, this could cause notifications to not display as unread. 

So many possibilites, even a bug. Hard to pinpoint exactly sorry. But, try the step above and see if it helps.

**Reply by MartinDelille** [reply]
Thanks for your answer! I followed your advice and keep you informed if it fix the issue!

**Comment by Leon406** [comment]
Solution from https://github.com/orgs/community/discussions/43032 has solved my problem

> Goto Done.
Select all
Move to inbox 
Goto Inbox
Select all
Mark as read
Mark as Done

**Reply by MartinDelille** [reply]
I tried that. Could you kindly reply to this thread so I check the unread badge appear as expected ?

**Reply by Leon406** [reply]
> I tried that. Could you kindly reply to this thread so I check the unread badge appear as expected ?

 Move **all done** notificatons to inbox, This opertion  only  can move 25 notificatons at one time, and don't refresh list
 you should refresh page and move rest  notificaton until  there is no notificaton

**Reply by MartinDelille** [reply]
It didn't worked indeed.

Unfortunately I have 2500 notifications. Is there a way to erase _Done_ notifications or batch the process ?

**Reply by MartinDelille** [reply]
After trying doing the job with the api but not finding how to move a notification to inbox programatically, I ended up marking as done all the notification and saving the one I want to track. There is really a bug with unread badge.

**Reply by ahuigo** [reply]
This works. Be aware that if you have multi-pages, after selecting the first page it will show a link to select the rest.

**Reply by FrankReh** [reply]
Problem started for me yesterday. So I moved a few sets of the 25 latest Done back to the Inbox and then from the Inbox could select them all and hit Done again. That got my count back to zero. Thanks for the tip.

**Reply by Alizter** [reply]
This workaround works but I've found myself having to do it multiple times this week. @github when will you fix this?
